consensus/src/iteration_ctx.rs

```diff
@@ -86,7 +88,7 @@ impl<D: Database> IterationCtx<D> {
         ratification_handler: Arc<
             Mutex<ratification::handler::RatificationHandler>,
         >,
-        round_base_timeout: Duration,
+        base_timeouts: TimeoutSet,
```

Comment:
This should not be called `base_timeouts` but `timeouts`.
Base timeouts are just the initial value for a new round. But then each iteration has its own timeout set, depending on what happened in the previous iteration